src/entities/course/ui/course-item/course-item.test.tsx

```diff
@@ -40,14 +40,6 @@ describe('CourseItem Component', () => {
     );
   });
 
-  it('renders the course image with correct src and alt attributes', () => {
```

Comment:
i deleted this test, because i thougth since we are using our `Image` component, all that is related for displaying the images must be tested in this component accordingly